schema.prisma

```diff
@@ -66,6 +66,20 @@ model Person {
   @@map("people")
 }
 
+model Bootcamp {
+  id     String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  status  String
+  title  String
+  email   String
+  message String
+  date  DateTime
+  firstName String?
+  lastName String?
+  @@map("bootcamp")
+}
+
```

Comment:
After modifying the `schema.prisma` file you need to follow the procedure of generating a migration and rebuilding the schema:

```
yarn prisma migrate dev

yarn generate-schema
```

With these steps your PR will be complete

Reply:
Thanks a lot!